src/clib/lib/python/enkf_defaults.cpp

```diff
@@ -1,7 +1,4 @@
 #include <ert/enkf/enkf_defaults.hpp>
 #include <ert/python.hpp>
 
-ERT_CLIB_SUBMODULE("enkf_defaults", m) {
-    m.add_object("DEFAULT_GEN_KW_EXPORT_NAME",
-                 py::str(DEFAULT_GEN_KW_EXPORT_NAME));
-}
+ERT_CLIB_SUBMODULE("enkf_defaults", m) {}
```

Comment:
Why keep this if it does nothing?

Reply:
I thought that it was added to expose things - but so far only one thing needed exposition.

i removed that one thing, but figured we should leave the general structure of exposing things in place, so we can expose things again easily when necessary, as so one doesn't need to figure out the pattern anew. but it's mostly due to me not being familiar with the pattern, and maybe for most other people it's not a big deal, and can easily be removed and later reproduced.

what do you say, toss it?

Reply:
toss it

Reply:
alrighty : D it is done! gonna push in a minute, as soon as all tests are fixed locally.